app/assets/stylesheets/views/_covid.scss

```diff
@@ -100,40 +100,6 @@
   }
 }
 
-.covid__highlight-link {
-  display: inline-block;
-  display: -ms-flexbox;
-  display: flex;
-  -ms-flex-align: center;
-  align-items: center;
-  box-sizing: border-box;
-  padding-left: 50px;
-  min-height: 45px;
-  max-width: 460px;
-  background-image: image-url("covid-arrow.png"); // PNG fallback for SVG
-  background-image: image-url("covid-arrow.svg");
-  background-repeat: no-repeat;
-  background-position: 0 50%;
-  background-size: 37px auto;
-
-  @include govuk-media-query($until: tablet) {
-    max-width: 410px;
-    margin-bottom: govuk-spacing(8);
-  }
-
-  @include govuk-media-query($from: desktop) {
-    // IE10/11 specific - vertical align in flexbox is ignored
-    // unless an explicit height is set
-    @media screen and (min-width:0\0) {
-      height: 45px;
```

Comment:
We should probably add this back in as it's causing issues on IE (screenshots below are IE10):
<img width="586" alt="Screenshot 2020-04-23 at 10 52 29" src="https://user-images.githubusercontent.com/4599889/80085917-d6739b80-8550-11ea-8591-25e2df590449.png">
<img width="650" alt="Screenshot 2020-04-23 at 10 52 42" src="https://user-images.githubusercontent.com/4599889/80085918-d70c3200-8550-11ea-89c6-97338b8fa6d9.png">
<img width="307" alt="Screenshot 2020-04-23 at 10 52 01" src="https://user-images.githubusercontent.com/4599889/80085907-d378ab00-8550-11ea-8a9b-a886bbcdb8ac.png">

Reply:
Ignore - this is an issue with the current live version not the new version